pool/pool.go

```diff
@@ -1527,8 +1527,6 @@ type resCreateSession struct {
 // Each method which produces a NeoFS API call may return an error.
 // Status of underlying server response is casted to built-in error instance.
 // Certain statuses can be checked using `sdkClient` and standard `errors` packages.
-// Note that package provides some helper functions to work with status returns
-// (e.g. sdkClient.IsErrContainerNotFound, sdkClient.IsErrObjectNotFound).
```

Comment:
I guess this hunk is in the wrong commit, methods are deleted by subsequent commits.

Reply:
Moved